### PR TITLE
Xch/test new nbgl

### PR DESCRIPTION
**Pull request by xchapron-ledger**
Demo of https://github.com/LedgerHQ/ledger-secure-sdk/pull/576

### CODE REVIEW

File: src/ui_nbgl.c [hunk 1]
```diff
@@ -249,12 +246,12 @@ void ui_display_action_sign_done(parserStatus_e status, bool validated) {
...
 
-    nbgl_useCaseReviewStart(&C_stax_app_eos_64px,
-                            "Review transaction",
-                            review_title,
-                            "Reject transaction",
-                            user_action_multipls_action_sign_flow_ok,
-                            reject_confirmation);
+    nbgl_useCaseReviewStreamingStart(TYPE_OPERATION,
+                       &C_stax_app_eos_64px,
+                       "Review operation",
```
Comment (sgliner-ledger) [inline src/ui_nbgl.c:255]: If we use `TYPE_OPERATION` can't both string be defined in the SDK? 
Maybe default behaviour could be to have them set to NULL, and if the developer want to customize he could still set it ?

File: src/ui_nbgl.c [hunk 2]
```diff
@@ -191,56 +171,73 @@ static nbgl_layoutTagValue_t* get_single_action_review_pair(uint8_t index) {
...
+static nbgl_layoutTagValue_t* get_multi_action_review_pair(uint8_t index) {
+    if (index == 0) {
+        pair.item = "Review action";
+        snprintf(review_title, sizeof(review_title), "%d of %d",
+                 txProcessingCtx.currentActionIndex,
+                 txProcessingCtx.currentActionNumer);
+        pair.value = review_title;
+        return &pair;
+    } else {
+        return get_single_action_review_pair(index -1);
```
Comment (sgliner-ledger) [inline src/ui_nbgl.c:188]: weird spacing

File: src/ui_nbgl.c [hunk 3]
```diff
@@ -191,56 +171,73 @@ static nbgl_layoutTagValue_t* get_single_action_review_pair(uint8_t index) {
...
+            user_action_single_action_sign_flow_ok();
+        }
     } else {
-        infoLongPress.text = "Accept & review next";
-        infoLongPress.longPressText = "Hold to continue";
+        user_action_tx_cancel();
     }
+}
 
+void ui_display_single_action_sign_flow(void) {
```
Comment (sgliner-ledger) [inline src/ui_nbgl.c:212]: Looking at the code, I find it not so easy to follow functions to understand what the final screens will look like. Maybe we should try to find a descriptive way to comment the screens, such as what used to be done on bagl ? :thinking:

File: src/ui_nbgl.c [hunk 4]
```diff
@@ -191,56 +171,73 @@ static nbgl_layoutTagValue_t* get_single_action_review_pair(uint8_t index) {
...
     pairList.nbMaxLinesForValue = 0;
-    pairList.nbPairs = txContent.argumentCount + 2;
     pairList.pairs = NULL; // to indicate that callback should be used
-    pairList.callback = get_single_action_review_pair;
     pairList.startIndex = 0;
 
-    nbgl_useCaseStaticReview(&pairList, &infoLongPress, "Reject transaction", review_choice);
-}
+    if (txProcessingCtx.currentActionNumer == 1) {
```
Comment (sgliner-ledger) [inline src/ui_nbgl.c:217]: numer :1234:

**Review by sgliner-ledger — DISMISSED** [review]

**Comment by cedelavergne-ledger** [comment]
Closing, because integrated, with latest NBGL fetures/API in https://github.com/LedgerHQ/app-eos/pull/28

**Pull request closed without merging**